doc/newsboat.asciidoc

```diff
@@ -726,6 +726,13 @@ See also <<miniflux-passwordfile,`miniflux-passwordfile`>>,
 <<miniflux-passwordeval,`miniflux-passwordeval`>>, and
 <<_passwords_for_external_apis>>.
 
+It is also recommended to explicitly specify the type of HTTP authentication
+that should be used when connecting to Miniflux, since Newsboat fails to guess
+the correct type on the first try, resulting in extra HTTP requests. This can be
+fixed by setting the following option in the configuration file:
+
+	http-auth-method basic
```

Comment:
Does Miniflux *require* that, or is it just how it's usually deployed? Can `MinifluxApi` make users' lives easier by defaulting to basic auth if `http-auth-method` is not set? That way, everything works seamlessly by default, but people who use different auth method can still configure Newsboat to act accordingly.

Reply:
In the interest of merging this PR sooner than later: if you try to implement the above check and run into any trouble, just let me know and I'll merge without it. The check can always be added later via a separate PR.

Reply:
That's a good question; Miniflux can also use API keys but I haven't implemented that so far since there's not a "generic" way to store secrets outside of the password{,file,eval} configs (although now that I mention it, maybe I should just add an additional config variable to specify that the "password" is actually an API key so we support both). I noticed the double-request behavior both with my personal instance and with the docker-compose instance, so I think it would probably be safe to auto-set the `http-auth-method` to basic and then add a note in the docs that non-standard configurations might need to explicitly change it.

Reply:
Went ahead and added some code to auto-set the HTTP auth method if it's "any" (default). I'm not sure though if this is the right place for it or if there might be a case where someone would want to force the "any" option (since this would always overwrite the option).

Reply:
> I'm not sure though if this is the right place for it or if there might be a case where someone would want to force the "any" option (since this would always overwrite the option).

Well, if someone needs that, they'll open an issue and we'll think what can be done.

Thanks for streamlining the happy path!